$NetBSD: patch-configure.ac,v 1.1 2019/12/17 13:59:50 nros Exp $

* check for getauxval and sys/auxv.h since it is not available on all systems

--- configure.ac.orig	2019-12-16 17:00:33.348861095 +0000
+++ configure.ac
@@ -103,7 +103,8 @@ AC_CHECK_TYPES(socklen_t, [], [])
 
 dnl check for getopt in standard library
 dnl AC_CHECK_FUNCS(getopt_long , , [LIBOBJS="$LIBOBJS getopt.o getopt1.o"] )
-AC_CHECK_FUNCS(getopt_long, [], [])
+AC_CHECK_FUNCS([getopt_long getauxval], [], [])
+AC_CHECK_HEADERS([sys/auxv.h])
 
 AC_CHECK_SIZEOF(void*,1)
 
